python/call_synthesize.py

```diff
@@ -37,8 +50,11 @@ def call_synthesize(args, text):
                 else:
                     ws.setFrameRate(response.audio.sample_rate_hertz)
                 ws.append(response.audio.content)
-                if response.audio.end_of_stream:
-                    ws.save(wavefilename)
+
+        if out_format == tribune_tts_pb2.OutputFormat.PCM16:
```

Comment:
You can set `out_format` in WaveSaver constructor and here do not use condition

Reply:
Done.